index.js

```diff
@@ -159,9 +201,13 @@ module.exports = {
 
     var fileExists = (
       this._doesFileExistInDummyApp(relativePath) ||
-      this._doesFileExistInCurrentProject(relativePath)
+      this._doesFileExistInCurrentProjectApp(relativePath) ||
+      this._doesFileExistInCurrentProjectAddon(relativePath) ||
+       this._doesFileExistIsCurrentProjectAddonModule(relativePath)
     );
 
+    console.log(!fileExists, name, relativePath);
```

Comment:
I think this is still in here; I'm seeing a bunch of console logging right now trying to run things on `ember-ajax` that match this pattern